src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -312,12 +363,37 @@ val qualityFormat: Option[FastqQualityFormat] = None,
     // Close the writer; NB: the inputs close automatically
     sampleInfos.foreach(_.writer.close())
 
+    // Optionally create the FASTQs
+    if (outputFastqs) {
+      logger.info("Writing FASTQs")
+      val fastqProgress = new ProgressLogger(this.logger, unit=1e6.toInt)
+      sampleInfos.foreach { info =>
+        val outputFastq    = PathUtil.replaceExtension(path=info.outputBam, ext=".fastq.gz")
+        val bufferedWriter = IOUtil.openFileForBufferedWriting(outputFastq.toFile)
+        val fastqWriter    = FastqWriter(bufferedWriter)
+        val bamReader      = SamReaderFactory.makeDefault().open(info.outputBam)
+        import com.fulcrumgenomics.FgBioDef.javaIterableToIterator
+        bamReader.foreach { bamRecord =>
+          val fastqRecord = FastqRecord(
```

Comment:
This appears to write all the reads to a single fastq file, without their read number suffixes.  Shouldn't this either output to a pair of fastq files, or supply `readNumber` below?